src/components/profile-view/profile-view.jsx

```diff
@@ -1,18 +1,28 @@
-import { useState } from "react";
+import { useState, useEffect } from "react";
 import { useNavigate } from "react-router-dom";
 import { Col, Row, Container } from "react-bootstrap";
 import { Button, Card, Form } from "react-bootstrap";
 import { MovieCard } from "../movie-card/movie-card";
 import { PersonSquare } from "react-bootstrap-icons";
-import moment from 'moment';
+// import moment from 'moment';
 import axios from "axios";
 import UserInfo from "./user-info";
+import FavoriteMovies from "./favorite-movies";
+import UserForm from "./user-form";
+import "./profile-view.scss";
```

Comment:
Looks like you have imported a bunch of components that not being used. Kindly ensure that everything import is used, else get rid of it because it impacts performance of the application.